### DISCUSSION TITLE
2.3.20 ISO link not working

### DISCUSSION THREAD

**Discussion by jrlane** [Security-Onion-Solutions/securityonion#2416] [General] [answered]
https://download.securityonion.net/file/securityonion/securityonion-2.3.20.iso

returns:

{
  "code": "not_found",
  "message": "File with such name does not exist.",
  "status": 404
}

**Comment by TOoSmOotH** [comment, marked as answer]
Yea we pulled it due to a bug. Testing .21 right now and will upload it when it has gone through thorough testing.

**Comment by TOoSmOotH** [comment]
2.3.21 is out there now
